packages/seacas/libraries/ioss/src/Ioss_ZoneConnectivity.C

```diff
@@ -84,6 +85,179 @@ namespace Ioss {
     return os;
   }
 
+#define IJK_list(v) v[0],v[1],v[2]
+#define IJK_format "%d:%d:%d"
```

Comment:
I don't think this is used.

Reply:
It's not.  And it's being used incorrectly in StructuredBlock.  I'll fix both.